### DISCUSSION TITLE
devProxy not override development server routes

### DISCUSSION THREAD

**Discussion by engenes** [nuxt/nuxt#16095] [Questions]
After update Nuxt from rc-11 to stable 3.0.0, my proxy stopped working.
This is the config fragment responsible for override in rc-11
```
...
  vite: {
    server: {
      proxy: {
        '/api': {
          target: 'http://project-local.app',
        },
      },
    },
....
```
But now it immediately returns 404
I found that the procs are now configured like this:

```
  nitro: {
    devProxy: {
      '/api': { target: 'http://project-local.app' },
    },
  },
```
I still get a bug, even though the request takes longer
```
Request URL: http://project-local.app:8080/api/property/sections
Request Method: GET
Status Code: 404 Not Found
Remote Address: 127.0.0.1:8080
Referrer Policy: strict-origin-when-cross-origin
```
http://project-local.app:8080 it's front url
http://project-local.app - back url

In the project branch with rc-11 everything works as usual.
Tell me where I'm wrong, please

**Comment by jb-alvarado** [comment]
I had similar issues and could fix it like:

```NGINX
nitro: {
    devProxy: {
      '/api': { target: 'http://project-local.app/api' },
    },
  }
```
